src/test/java/org/apache/drillui/test/framework/pages/ProfilesPage.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.drillui.test.framework.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class ProfilesPage extends BasePage {
+  @FindBy(css = "tbody>tr[role='row']>td>a")
+  private List<WebElement> queryResult;
+
+  @FindBy(xpath = "//tbody/tr[@role='row']/td[2]")
+  private List<WebElement> ProfileNamesList;
+
+  @FindBy(css = "select[name='profileList_completed_length']")
+  private WebElement dropDownList;
+
+  @FindBy(css = "select[name='profileList_completed_length']>option[value='-1']")
+  private WebElement ListItem_All;
+
+  public void SelectAllInDropDownList(){
+    dropDownList.click();
+    ListItem_All.click();
+  }
+
+  public boolean isQueryProfileINList(String queryProfile) {
+    for (WebElement i : queryResult) {
+      if (i.getAttribute("href").contains(queryProfile)) return true;
+    }
+    return false;
+  }
+
+  public boolean isUsersNameInProfileList(String userName){
+    for (WebElement i : ProfileNamesList) {
+      if(i.getText().equals(userName)){
+        continue;}
+      else return false;
```

Comment:
It's better to move "}" bracket from "if" statement to the new line before the "else" keyword 
```suggestion
      if(i.getText().equals(userName)) {
        continue;
      } else return false;
```

Reply:
Done

Reply:
It's better to use brackets everywhere:

```Java
if(i.getText().equals(userName)) {
  continue;
} else {
  return false;
}
```

Reply:
Done